### ISSUE TITLE
[C#] Visual Studio wrongly reports IDE0052 ("get accessor is never invoked")

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#52653]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/C-Visual-Studio-wrongly-reports-IDE00/1396219)._

---
[severity:It bothers me. A fix would be nice]
Consider the following code:
```
class Test
{
    int X { get => _x; set => _x = value + 1; }
    int _x;

public void DoTest()
    {
        X++; //equivalent to X = X + 1, thus the get accessor must be invoked in this line
    }
}
```
Visual studio marks the X property with the IDE0052 suggestion/warning:
**Private property 'Test.X' can be converted to a method as its get accessor is never invoked.**

This is wrong, because the get accessor is in fact invoked in the method "DoTest()".
Replacing `X++` with `X=X+1` removes the warning, but in turn triggers the IDE0054 warning: **Use '++' operator.**

---
#### Feedback Bot on 4/13/2021, 07:42 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by CyrusNajmabadi** [comment]
@mavasani ?

**Comment by mavasani** [comment]
Yep, seems like an FP. Marked as help wanted.

**Comment by Youssef1313** [comment]
Duplicate of #43191.
